src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/SecondarySite/index.js

```diff
@@ -101,71 +102,75 @@ const StepPrimarySetup = () => {
 						) }
 					/>
 				</div>
-
-				<div className="nfd-setup-secondary-categories">
-					<div className="nfd-card-category-wrapper">
-						<div className="category-scrolling-wrapper">
-							<span
-								className="icon"
-								style={ {
-									backgroundImage: categoriesArray[ 0 ].icon,
-								} }
-							/>
-							<p className="categName">
-								{ ' ' }
-								{ categoriesArray[ 0 ].name }
-							</p>
+				<StepStateHandler watch={ categoriesArray[ 0 ]?.subCategories }>
```

Comment:
Let's also wait on `chosenSecondaryCategory` for the same reason

Reply:
added this "selectedCategoryInStore" also to the check ... "chosenSecondaryCategory" is the CSS class name